xds/internal/xdsclient/loadreport.go

```diff
@@ -29,5 +32,11 @@ import "google.golang.org/grpc/xds/internal/xdsclient/load"
 // It returns a Store for the user to report loads, a function to cancel the
 // load reporting stream.
 func (c *clientImpl) ReportLoad(server string) (*load.Store, func()) {
-	return c.controller.ReportLoad(server)
+	// TODO: load reporting with federation needs special handling. This is to
```

Comment:
Do we already know what special handling this needs? If so, does it make sense to mention it here?

Reply:
Done.

I also noticed it doesn't correctly unref the authority. Fixed.